libqtile/backend/wayland/xwindow.py

```diff
@@ -393,10 +406,20 @@ def _on_unmap(self, _listener: Listener, _data: Any) -> None:
         self.core.pending_windows.add(win)
 
     def _on_commit(self, _listener: Listener, _data: Any) -> None:
-        logger.debug("Signal: xstatic commit")
+        pass
+        # logger.debug("Signal: xstatic commit")
 
     def _on_request_configure(self, _listener: Listener, event: SurfaceConfigureEvent) -> None:
         logger.debug("Signal: xstatic request_configure")
+        cw = ConfigWindow
```

Comment:
The wayland backend does not depend on xcffib, and we shouldn't make it. If we just need these 4 constants, could we just lift them and stick them into this file?

Reply:
Yes, that's easy enough.  They're just 1, 2, 4, and 8, respectively :)